metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/topology/ParserTopologyBuilder.java

```diff
@@ -132,7 +132,14 @@ public static TopologyBuilder build(String zookeeperUrl,
     if(securityProtocol.isPresent()) {
       kafkaSpoutConfigOptions.putIfAbsent("security.protocol", securityProtocol.get());
     }
-    return SimpleStormKafkaBuilder.create(inputTopic, zkQuorum, Arrays.asList("value"), kafkaSpoutConfigOptions);
+    return SimpleStormKafkaBuilder.create( inputTopic
```

Comment:
This seems likely to have conflict.  The assumption that these simple names may never already be a part of a message.  In general, the metadata is nebulous.  The metadata fields should be prefixed and discoverable by other processes, without knowing before hand.

Some call Map<?,?> getMetadata(JSONObject message) should be implementable without hard coding.